modules/help_editor/jsx/helpEditorForm.js

```diff
@@ -0,0 +1,168 @@
+import swal from 'sweetalert2';
+import React, {useState, useEffect} from 'react';
+import {unmountComponentAtNode, createPortal} from 'react-dom';
+import Help from 'jsx/Help';
+
+/**
+ * Help Editor Form Page.
+ *
+ * Serves as an entry-point to the module, rendering the whole react
+ * component page on load.
+ *
+ * Renders Help Editor form page
+ *
+ * @author LORIS Team
+ * @version 1.0.0
+ */
+
+const HelpEditorForm = (props) => {
+  const [title, setTitle] = useState(props.title ?? '');
+  const [content, setContent] = useState(props.content ?? '');
+
+  const returnLabel = 'Return to ' + props.module;
+  const helpContainers = document.getElementsByClassName('help-container');
+
+  useEffect(() => {
+    Array.from(helpContainers).forEach((helpContainer) =>
+      unmountComponentAtNode(helpContainer)
```

Comment:
When I googled for this it says it's being replaced by "root.unmount()". Seems like it's already deprecated.

Reply:
Yes, I saw that too.
`In React 18, unmountComponentAtNode was replaced by [root.unmount()](https://beta.reactjs.org/reference/react-dom/client/createRoot#root-unmount).`
We are using react 17 so I'm not sure if root.unmount() is already available.

Reply:
Maybe it would be best to rebase on #8188 and use it

Reply:
Yes, that's a good plan! I can also replace the other deprecated functions (ReactDOM.render, ...).